package.json

```diff
@@ -1,24 +1,25 @@
 {
-	"name": "bup",
-	"author": "Philipp Hagemeister <phihag@phihag.de>",
-	"description": "Badminton Umpire Panel",
-	"repository": {
-		"type": "git",
-		"url": "git://github.com/phihag/bup.git"
-	},
-	"scripts": {
-		"test": "mocha test/",
-		"test:watch": "npm run test -- -w"
-	},
-	"dependencies": {
-	},
-	"devDependencies": {
-		"async": "*",
-		"eslint": "*",
-		"jshint": "*",
-		"mocha": "*",
-		"parse-appcache-manifest": "*",
-		"ws": "*"
-	},
-	"private": true
+  "name": "bup",
```

Comment:
That was npm's fault. Whenever `package.json` is touched by npm (even with `npm install`),  npm changes the indentation to two spaces. Should have changed the indentation back to tabs before committing...